main.js

```diff
@@ -0,0 +1,94 @@
+const userName = document.getElementById("name");
+const userPost = document.getElementById("post");
+const submit = document.getElementById("submit");
+const posts = document.getElementsByClassName("posts")[0];
+
+submit.addEventListener("click", onSubmit);
+
+function onSubmit() {
+  if (userName.value === "" || userPost.value === "") {
```

Comment:
this is a nice try  for validation, but actually you can write spaces in the input and this wont trigger the alert and then you have an empty post
![image](https://github.com/user-attachments/assets/1883e040-d597-42cd-b46f-865cc3034847)